src/main/java/io/kestra/plugin/ai/completion/Classification.java

```diff
@@ -39,32 +43,39 @@
             full = true,
             code = {
                 """
-                id: text_categorization
-                namespace: company.ai
-
-                tasks:
-                  - id: categorize
-                    type: io.kestra.plugin.ai.completion.Classification
-                    prompt: "Categorize the sentiment of: I love this product!"
-                    classes:
-                      - positive
-                      - negative
-                      - neutral
-                    provider:
-                      type: io.kestra.plugin.ai.provider.GoogleGemini
-                      apiKey: "{{ kv('GEMINI_API_KEY') }}"
-                      modelName: gemini-2.5-flash
-                """
+                    id: text_categorization
+                    namespace: company.ai
+
+                    tasks:
+                      - id: categorize
+                        type: io.kestra.plugin.ai.completion.Classification
+                        messages:
+                          - type: SYSTEM
+                            content: You are a sentiment analysis assistant. Classify text as positive, negative or neutral.
+                          - type: USER
+                            content: "I absolutely love this product, it's fantastic!"
+                        classes:
+                          - positive
+                          - negative
+                          - neutral
+                        provider:
+                          type: io.kestra.plugin.ai.provider.GoogleGemini
+                          apiKey: "{{ kv('GEMINI_API_KEY') }}"
+                          modelName: gemini-2.0-flash
+                    """
             }
         )
     },
     aliases = {"io.kestra.plugin.langchain4j.Classification", "io.kestra.plugin.langchain4j.completion.Classification"}
 )
 public class Classification extends Task implements RunnableTask<Classification.Output> {
 
-    @Schema(title = "Text prompt", description = "The input text to classify")
+    @Schema(
+        title = "Chat Messages",
+        description = "The list of chat messages for the current conversation. There can be only one system message, and the last message must be a user message."
+    )
     @NotNull
-    private Property<String> prompt;
```

Comment:
Please keep a single prompt.
The idea was to use a system message under the cover to add the classification instruction instead of concatenating them inside the user prompt.

Another way to do it would be to add the instruction as default value in a `systemMessage` so the user could update the instruction if he want.